app/services/achiever/course/template.rb

```diff
@@ -156,18 +160,7 @@ def self.find_many_by_stem_course_template_nos(stem_course_template_nos)
   end
 
   def by_certificate(certificate)
-    case certificate
-    when "subject-knowledge"
-      @programmes.include?("Subject Knowledge")
-    when "secondary-certificate"
-      @programmes.include?("Secondary")
-    when "primary-certificate"
-      @programmes.include?("Primary")
-    when "i-belong"
-      @programmes.include?("I Belong")
-    when "a-level-certificate"
-      @programmes.include?("A Level")
-    end
+    @programmes.include?(certificate)
```

Comment:
Does this need to go through the mapping?

Reply:
Ahhh! no, I see what is going on. You've mapped from the Title Case to slugs right at the very start. Nice

Reply:
No, as we are remapping the programme code from dynamic to the slug, and the filter on the course page passes the slug as its value, so there is no need to map backwards